html/js/tree/Move.js

```diff
@@ -0,0 +1,34 @@
+GTE.TREE = (function (parentModule) {
+    "use strict";
+
+    /**
+    * Creates a new Move.
+    * @class
+    * @param {Node} parent Parent node origin of the move.
+    * @param {Node} child Child node destiny of the move.
```

Comment:
r/destiny/destination
 
http://forum.wordreference.com/threads/destiny-and-destination.285244/?hl=es